examples/cross-contract-call-tracing/lib.rs

```diff
@@ -142,14 +142,17 @@ mod tests {
 
     #[test]
     fn test() -> Result<(), Box<dyn Error>> {
-        let mut session = Session::<MinimalRuntime>::new(Some(transcoder()))?;
+        let mut session = Session::<MinimalRuntime>::new()?;
         session.override_debug_handle(DebugExt(Box::new(TestDebugger {})));
 
-        let outer_address = session.deploy(bytes(), "new", NO_ARGS, vec![1], None)?;
+        let outer_address =
+            session.deploy(bytes(), "new", NO_ARGS, vec![1], None, &transcoder())?;
```

Comment:
I'm wondering, for purely aesthetic reasons, whether the optional argument should come last.

Reply:
actually in rust there are no optional arguments, just a name coincidence :P I was grouping these arguments by the context they are used; not sure if that's the most convenient order - I can make transcoder as the first argument, which will accidentally make storage deposit coming last :P